midi_app_controller/gui/binds_editor.py

```diff
@@ -438,9 +536,9 @@ def _switch_editors(self, checked: bool):
 
     def _toggle_names_mode(self):
         """Toggles actions names mode: titles or ids."""
-        for _, combo in self.buttons_widget.button_combos:
+        for _, combo in self.buttons_widget.button_combos.items():
             combo.toggle_names_mode()
-        for _, combo1, combo2 in self.knobs_widget.knob_combos:
+        for _, combo1, combo2 in self.knobs_widget.knob_combos.items():
```

Comment:
same as above, should be `for _, (combo1, combo2)`. Or `for combo1, combo2 in ...values()`